packages/ffe-form-react/src/HorizontalSelector.spec.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { mount } from 'enzyme';
+// import jest from 'jest';
+import HorizontalSelector from './HorizontalSelector';
+
+const options = [
+    {
+        value: 0,
+        label: '0',
+    },
+    {
+        value: 1,
+        label: '1',
+    },
+    {
+        value: 2,
+        label: '2',
+    },
+    {
+        value: 3,
+        label: '3',
+    },
+    {
+        value: 4,
+        label: '4',
+    },
+];
+const getWrapper = () => {
+    return mount(
+        <HorizontalSelector
+            options={options}
+            selected={options[0]}
+            label={'Antall barn'}
+            description={'Velg hvor mange barn du har'}
+            onChange={() => {}}
+            field={'antallbarn'}
+        />,
+    );
+};
+
+describe('<HorizontalSelector>', () => {
+    const optionClassName = 'horizontal-selector-option';
+    const elementByCSS = `.${optionClassName}`;
+    const selectedClassName = `${optionClassName} horizontal-selector-selected horizontal-selector-option--text-center`;
+    it('renders the appropriate number options', () => {
+        const wrapper = getWrapper();
+        expect(wrapper.exists()).toBe(true);
+        expect(wrapper.find('.horizontal-selector').find('label').length).toBe(
+            options.length,
+        ); // hasClass('ffe-field-error-message')).toBe(true);
+    });
+    it('sets the correct selected option when clicked', () => {
+        const wrapper = getWrapper();
+        expect(wrapper.exists()).toBe(true);
+
+        let initialSelectedElement = wrapper.find(elementByCSS).at(0);
+        expect(initialSelectedElement.hasClass(selectedClassName)).toBe(true);
+
+        const element = wrapper.find(elementByCSS).at(3);
```

Comment:
typiskt dålig att bruka css for og finne ett element. Det er en implemntasjonsdetalj.

Reply:
Class navnet er jo knyttet opp mot komponentens struktur. Så den skal funke så lenge markupen er lik.

Hva mener du er den beste måten å finne et element på i denne sammenhengen ?

Reply:
classnavnet er en implementation detail. 

Denne er lesverd: https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change

Jag bruker selv 
https://github.com/testing-library/react-testing-library

Her hade jag nok brukt ByLabelText

Reply:
Skriv gjerne testene med `react-testing-library` fremfor `enzyme` ja.
Det gjør det mye lettere å teste _funksjon_ uten å teste eller være avhengig av implementasjonsdetaljer.

Reply:
Jeg bare så på de andre komponentene og eksemplet som ble lenket til designsystem chatten. Der brukes jo enzyme. 

Mye meninger her. 
Jeg må bare spørre, hvordan skal folk vite dette og hvem bestemmer ?

Reply:
react-testing-library har nyligen blitt inført. Det er ikke jag som bestemmer noe som helst vad man bruker men har spurt @kwltrs og @antidecaf utan att få noen protester. Spurte også på Slack kanalen. Problemet med Enzyme er att det tillåter deg att skriva dåliga tester, man tvingas ikke det det men mye enklare. shallow render, getIntance... bara implementasjonsdetaljer som testas.

Så før meg bruk vad du vill, æven om jag selv valgt react-testing-library.

På alla nya prosjekt i spareteamet kør vi react-testing-library. På gamla har vi det sida vid sida sammen med Enzyme akkurat som her.